app/src/date-utils.ts

```diff
@@ -389,27 +389,40 @@ const DateUtils = {
    *
    * The returned date/time format depends on how long ago the timestamp is.
    */
-  shortTimeString(datetime) {
+  shortTimeString(datetime: Date) {
```

Comment:
This all looks great, thank you for updating it to use the new browser standard APIs!